app/jobs/batch_create_job.rb

```diff
@@ -28,7 +28,7 @@ def create(user, titles, resource_types, uploaded_files, attributes, log)
       raise ArgumentError, 'attributes must include "model" => ClassName.to_s' unless model
       uploaded_files.each do |upload_id|
         title = [titles[upload_id]] if titles[upload_id]
-        resource_type = [resource_types[upload_id]] if resource_types[upload_id]
+        resource_type = Array(resource_types[upload_id]) if resource_types[upload_id]
```

Comment:
Favor `Array.wrap` instead of `Array`.

```ruby
 Array(Time.now)
=> [19, 15, 9, 23, 1, 2017, 1, 23, false, "EST"]

Array(Time.now)
=> [Time.now]
```